src/main/scala/nl.knaw.dans.easy.stage/Settings.scala

```diff
@@ -85,6 +89,8 @@ object Settings {
       databaseUrl = databaseUrl,
       databaseUser = databaseUser,
       databasePassword = databasePassword,
-      licenses = licenses)
+      licenses = licenses,
+      stageDatasetVersion = stageDatasetVersion
```

Comment:
to avoid future diffs looking nasty 😉

```suggestion
      stageDatasetVersion = stageDatasetVersion,
```